Utilities/DataFlow/test/test_SubframeUtils01.cxx

```diff
@@ -0,0 +1,56 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
There was no formal evaluation. I remember a discussion in CWG11 about it. We went for boost.test because any ways we depend on boost. Using gtest meant adding a dependency. As far as I remember, that was the main argument (?).

Pragmatically, at this stage, I would be in favour of continuing with boost.test just because all the tests are using it (also in the other O2 repos such as flp proto). If someone is motivated to make the formal evaluation though, let's do it. If we decide to change there would be then some work to modify the CMake macro, the recipes and the existing tests.